cmd/tnf/generate/catalog/catalog.go

```diff
@@ -157,19 +256,28 @@ func outputTestCaseBuildingBlocks() {
 	// Sorting the map by identifier URL
 	sort.Strings(keys)
 
-	// Iterating the map by sorted identifier URL
-	for _, k := range keys {
-		fmt.Fprintf(os.Stdout, "### %s", identifier.Catalog[k].Identifier.URL)
-		fmt.Println()
-		fmt.Println("Property|Description")
-		fmt.Println("---|---")
-		fmt.Fprintf(os.Stdout, "Version|%s\n", identifier.Catalog[k].Identifier.SemanticVersion)
-		fmt.Fprintf(os.Stdout, "Description|%s\n", identifier.Catalog[k].Description)
-		fmt.Fprintf(os.Stdout, "Result Type|%s\n", identifier.Catalog[k].Type)
-		fmt.Fprintf(os.Stdout, "Intrusive|%t\n", identifier.Catalog[k].IntrusionSettings.ModifiesSystem)
-		fmt.Fprintf(os.Stdout, "Modifications Persist After Test|%t\n", identifier.Catalog[k].IntrusionSettings.ModificationIsPersistent)
-		fmt.Fprintf(os.Stdout, "Runtime Binaries Required|%s\n", cmdJoin(identifier.Catalog[k].BinaryDependencies, ", "))
-		fmt.Println()
+	catalog := createPrintableCatalogFromUrls(keys)
+	if catalog == nil {
+		return
+	}
+
+	for i := 0; i < len(catalog); i++ {
```

Comment:
I don't think you use `i` here correct?  Is this additional for-loop needed?

Reply:
I don't think you need this at all if you are already range'ing through the `catalog`.

Reply:
yeah, you are right. This is because I am adding to each testName an array of identifiers. But, looking to the package, the test cases building blocks, each test has only one identifier.